corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -711,14 +711,25 @@ def _assert_reports_counts(self, domain_name, count):
         self._assert_queryset_count([
             ReportsSidebarOrdering.objects.filter(domain=domain_name),
             TableauServer.objects.filter(domain=domain_name),
-            TableauVisualization.objetcs.filter(domain=domain_name),
+            TableauVisualization.objects.filter(domain=domain_name),
         ], count)
 
     def test_reports_delete(self):
         for domain_name in [self.domain.name, self.domain2.name]:
             ReportsSidebarOrdering.objects.create(domain=domain_name)
-            TableauServer.objects.create(domain=domain_name)
-            TableauVisualization.objects.create(domain=domain_name)
+            server = TableauServer.objects.create(
+                domain=domain_name,
+                server_type='server',
+                server_name='my_server,',
```

Comment:
nitpick: remove the comma? probably doesn't matter for the test, it's just weird to see